modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -6,65 +6,64 @@
 [id="nw-ovn-kubernetes-matrix_{context}"]
 = Supported network plugin feature matrix
 
-{openshift-networking} offers two options for the network plugin, OpenShift SDN and OVN-Kubernetes, for the network plugin. The following table summarizes the current feature support for both network plugins:
+{openshift-networking} offers OVN-Kubernetes as the network plugin. The following table summarizes the current feature support for both network plugins:
 
 .Default CNI network plugin feature comparison
 [cols="50%,25%,25%",options="header"]
```

Comment:
```suggestion
[cols="50%,25%",options="header"]
```
Not sure about this, but you might need to remove this since you removed the third column.